src/Color/Scale.purs

```diff
@@ -53,19 +64,26 @@ stopColor :: ColorStop -> Color
 stopColor (ColorStop c _) = c
 
 -- | A color scale.
-data ColorScale = ColorScale ColorSpace Color (List ColorStop) Color
+data ColorScaleBuilder = ColorScaleBuilder Color (List ColorStop) Color
```

Comment:
Exporting this is very useful so users could write their own helper functions, without waiting for their PR to be merged. Before I needed to define my own type for it [see this](https://github.com/safareli/purescript-colorpalettepicker-halogen/blob/56a98be084c35e596438187aa1fc0152e5d8bb9a/src/ColorPalettePicker/Utils/PreScale.purs#L10)

Also if we have separate type which does not contain `ColorSpace` then we can implement `combineScale :: Number → Scale → Scale → Scale` which i did in the link above. If the Scale contained colorSpace then we would have to discard one of them, which I think will not be a good thing to do.

Reply:
:+1:

Reply:
What about just naming this `ColorStops` or `ColorStopList`?